pkg/core/state/tokens.go

```diff
@@ -62,7 +62,9 @@ type TokenTransferInfo struct {
 // NewTokenTransferInfo returns new TokenTransferInfo.
 func NewTokenTransferInfo() *TokenTransferInfo {
 	return &TokenTransferInfo{
-		LastUpdated: make(map[int32]uint32),
+		NewNEP11Batch: true,
+		NewNEP17Batch: true,
```

Comment:
I would prefer inverted `DirtyNEP11Batch` flag to better handle defaults.

Reply:
Makes sense, but can be done later.